app/views/tasks/show.html.slim

```diff
@@ -1,632 +1,621 @@
+        span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
+        span.fa.fa-caret-down.my-caret
+      .comment-box data-task=@task.id
+        = form_for :comment, url: task_comments_path(@task), remote: true do |f|
+          .write-comment
+            .input-group
+              = f.label 'new_comment_label', 'New Comment', class: 'btn btn-light disabled input-group-addon'
+              = f.text_field :text, class: 'form-control'
+              = f.submit class: 'btn btn-light'
+        .comment-body
 
-/.history-panel
-  = button_tag type: 'button', class: 'full-window-button history-button' do
-    => t('exercises.all.history_exercises')
-    span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
-    span.fa.fa-caret-down.my-caret
-/div class=("history-box with-borders") data-exercise=@task.id
-  .content
+    /.history-panel
+      = button_tag type: 'button', class: 'full-window-button history-button' do
+        = t('exercises.all.history_exercises')+ ' '
+        span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
+        span.fa.fa-caret-down.my-caret
+    /div class=("history-box with-borders") data-exercise=@task.id
+      .content
 
-.actions.btn-group[role="group"]
-  - if can? :edit, @task
-    = link_to edit_task_path(@task),  class: 'btn btn-important' do
-      i.fa.fa-edit
-      =< t('tasks.show.edit')
-  - if can? :destroy, @task
-    = link_to task_path(@task), class:'btn btn-important', method: :delete, data: { confirm: t('sure') } do
-      i.fa.fa-trash
-      =< t('tasks.show.delete')
-  /- if can? :report, @task
-    /.btn-group
-      = button_tag type: 'button', data: {toggle: "dropdown"}, aria: {haspopup: "true", expanded: "false"}, class: 'btn btn-important nav-btn-exercise dropdown-toggle dropdown-toggle split' do
-        => t('exercises.show.report')
-        span.caret
-      .dropdown-menu#share-menu
-        .dropdown-header style="padding-left: 2px;"
-          = t('exercises.show.report_message')
-          | :
-          .input-group
-            = text_field_tag 'text', nil, form: 'report', class: 'form-control'
-            .input-group-btn
-              = submit_tag t('exercises.show.submit'), form: 'report', class: 'btn btn-default'
-  = link_to duplicate_task_path(@task), method: :post, class: 'btn btn-important' do
-    i.fa.fa-clone
-    =< t('tasks.show.duplicate')
-  = link_to t('tasks.show.back'), tasks_path, class: 'btn btn-important'
+    .actions.btn-group[role="group"]
+      - if can? :edit, @task
+        = link_to edit_task_path(@task),  class: 'btn btn-important' do
+          i.fa.fa-edit
+          = ' ' + t('exercises.show.edit')
+      - if can? :destroy, @task
+        = link_to task_path(@task), class:'btn btn-important', method: :delete, data: { confirm: t('sure') } do
+          i.fa.fa-trash
+          = ' ' + t('exercises.show.delete')
```

Comment:
This change is unintended.

Reply:
Same as above: The translation starts with 'tasks' rather than 'exercise'. At least, that's the current status in the master branch.